<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScoreEditorDlg</class>
 <widget class="QDialog" name="ScoreEditorDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SMS Tools 2 - SMS Transformation Score Editor</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout">
   <item>
    <layout class="QVBoxLayout">
     <item>
      <widget class="QLabel" name="mScoreTitle">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Score</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Q3Frame" name="mScoreFrameContainer">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <layout class="QHBoxLayout">
        <item>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QLabel" name="mAvailableTransformationsLabel">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Available SMS Transformations</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Q3ListBox" name="mAvailableTransformationList">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="mAddToScoreBtn">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Add &amp;transformation to score</string>
            </property>
            <property name="shortcut">
             <string>Alt+T</string>
            </property>
            <property name="autoDefault">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QLabel" name="mScoreTransformationLabel">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Transformations in the Score</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Q3ListBox" name="mScoreTransformationList">
            <property name="minimumSize">
             <size>
              <width>175</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="mMoveUpBtn">
              <property name="minimumSize">
               <size>
                <width>65</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Move &amp;up</string>
              </property>
              <property name="shortcut">
               <string>Alt+U</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="mMoveDownBtn">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Move &amp;down</string>
              </property>
              <property name="shortcut">
               <string>Alt+D</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="mRemoveBtn">
              <property name="minimumSize">
               <size>
                <width>65</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>&amp;Remove</string>
              </property>
              <property name="shortcut">
               <string>Alt+R</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <item>
        <spacer name="spacer1">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>81</width>
           <height>21</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="mOkBtn">
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mCancelBtn">
         <property name="text">
          <string>Cancel</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout">
     <item>
      <widget class="QLabel" name="mTransformationLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Transformation</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Q3Frame" name="frame4">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QTabWidget" name="mTabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Description</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Parameters</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ListBox</class>
   <extends>Q3Frame</extends>
   <header>q3listbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mAvailableTransformationList</tabstop>
  <tabstop>mAddToScoreBtn</tabstop>
  <tabstop>mScoreTransformationList</tabstop>
  <tabstop>mMoveUpBtn</tabstop>
  <tabstop>mMoveDownBtn</tabstop>
  <tabstop>mRemoveBtn</tabstop>
  <tabstop>mTabWidget</tabstop>
  <tabstop>mOkBtn</tabstop>
  <tabstop>mCancelBtn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>mOkBtn</sender>
   <signal>clicked()</signal>
   <receiver>ScoreEditorDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mCancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>ScoreEditorDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mAddToScoreBtn</sender>
   <signal>clicked()</signal>
   <receiver>ScoreEditorDlg</receiver>
   <slot>addHighlightedToScore()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mMoveUpBtn</sender>
   <signal>clicked()</signal>
   <receiver>ScoreEditorDlg</receiver>
   <slot>moveHighlightedUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mMoveDownBtn</sender>
   <signal>clicked()</signal>
   <receiver>ScoreEditorDlg</receiver>
   <slot>moveHighlightedDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mRemoveBtn</sender>
   <signal>clicked()</signal>
   <receiver>ScoreEditorDlg</receiver>
   <slot>removeHighlightedFromScore()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mAvailableTransformationList</sender>
   <signal>highlighted(QString)</signal>
   <receiver>ScoreEditorDlg</receiver>
   <slot>onSelectAvailableTransformation(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mScoreTransformationList</sender>
   <signal>highlighted(int)</signal>
   <receiver>ScoreEditorDlg</receiver>
   <slot>onSelectTransformationInScore(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
